### PR TITLE
feat : Implemented Inverse Fast Fourier Transform

### PR DESCRIPTION
**Pull request by ameyachawlaggsipu**
Slightly different from fast Fourier transform
Just the om variable declared in line 40 is divided by n
and swapping the testing inputs with testing outputs .

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1834"></a>

### CODE REVIEW

File: numerical_methods/Inverse_fast_fourier_transform.cpp [hunk 1]
```diff
@@ -0,0 +1,161 @@
+/**
+ * @file
+ * @brief [An  inverse fast Fourier transform
```
Comment (Panquesito7) [inline numerical_methods/Inverse_fast_fourier_transform.cpp:3]:
```suggestion
 * @brief [An inverse fast Fourier transform
```

File: numerical_methods/Inverse_fast_fourier_transform.cpp [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+ * @brief Main function
+ * @param argc commandline argument count (ignored)
+ * @param argv commandline array of arguments (ignored)
+ * calls automated test function to test the working of fast fourier transform.
+ * @returns 0 on exit
+ */
+
+int main(int argc, char const *argv[]) {
+    test();  //  run self-test implementations
+              //  with 2 defined test cases
```
Comment (Panquesito7) [inline numerical_methods/Inverse_fast_fourier_transform.cpp:159]:
```suggestion
    test();  //  run self-test implementations
             //  with 2 defined test cases

```

File: numerical_methods/Inverse_fast_fourier_transform.cpp [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    return 0;
+}
```
Comment (Panquesito7) [inline numerical_methods/Inverse_fast_fourier_transform.cpp:161]:
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Great work! 😄👍
Please enable [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions) in your repository of this fork in this link: https://github.com/ameyachawlaggsipu/C-Plus-Plus/actions

File: numerical_methods/inverse_fast_fourier_transform.cpp [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+ * @param argc commandline argument count (ignored)
+ * @param argv commandline array of arguments (ignored)
+ * calls automated test function to test the working of fast fourier transform.
+ * @returns 0 on exit
+ */
+
+int main(int argc, char const *argv[]) {
+    test();  //  run self-test implementations
+             //  with 2 defined test cases
+
```
Comment (Panquesito7) [inline numerical_methods/inverse_fast_fourier_transform.cpp:160]:
```suggestion
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by amino19 — APPROVED** [review]

**Comment by ameyachawlaggsipu** [comment]
Thank You
@ayaankhan98 
@Panquesito7

**Pull request merged**